src/Input/Select/Select.tsx

```diff
@@ -207,6 +215,9 @@ class Select extends React.Component<Props, State> {
         const itemValue = activeElement.dataset.value;
         onChange(itemValue);
       }
+      if (onOptionClick !== undefined) {
+        onOptionClick();
```

Comment:
Maybe we can pass `event` as the parameter as we do in `handleClick`. Just come up with a scenario is devs can know the `onOptionClick` is triggered by keyboard or mouse via the `event` parameter.